tpm/tpm_test.go

```diff
@@ -498,3 +498,15 @@ func TestTakeOwnership(t *testing.T) {
 		t.Fatal("Couldn't take ownership of the TPM:", err)
 	}
 }
+
+func TestForceClear(t *testing.T) {
+	// Only enable this if you know what you're doing.
```

Comment:
Please add this descriptive comment and warning to the Skip statement to make it clear that users generally will not want to enable this.

Reply:
Done